source/sync/go-to-production/compact-disk.txt

```diff
@@ -4,62 +4,77 @@
 Compact an Atlas Volume
 =======================
 
-.. default-domain:: mongodb
+.. meta::
+  :description: Compact an Atlas volume to reclaim storage space for Atlas Device Sync data.
+
+.. facet::
+  :name: genre
+  :values: tutorial
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
+Atlas Device SDK's :ref:`data compaction <backend-compaction>` defragments the storage
+space the local database file used on a client and reduces the size of the
+``__realm_sync`` database on Atlas clusters. However, it does *not* reduce the
+overall size of the Atlas cluster. Compacting the data on an Atlas volume
+involves additional steps.
 
-Overview
---------
-
-While Realm :ref:`data compaction <backend-compaction>` defragments the storage 
-space the realm file uses on a *client*, and also reduces the size of the 
-``__realm_sync`` database on Atlas clusters, it does not reduce the overall size of 
-the Atlas cluster. Compacting the data on an Atlas volume involves several 
-additional steps. 
-
-Steps to Compact and Reclaim Disk Space
+Enable Compaction to Reclaim Disk Space
 ---------------------------------------
 
-You should use the following guidance for reducing Atlas disk space usage:  
+Perform the following to reduce Atlas disk space usage:
 
 1. Enable Compaction in App Services
 
-   Before reclaiming volume space, be sure that you have enabled Compaction 
-   on your Atlas App Services app. You set this by setting the Max Offline Time 
-   setting in the Atlas UI. For more information, see 
-   :ref:`<optimize-sync-atlas-usage>`.
+   Before reclaiming volume space, be sure that you have enabled Compaction
+   on your Atlas App Services app. To do so, set the :guilabel:`Client Max Offline Time`
+   in the Sync Settings through the App Services UI. For more information, refer to
+   :ref:`set-the-client-maximum-offline-time`.
 
    .. note:: Wait for Compaction
-   
-      If you are enabling compaction for the first time, you will need to wait for 
-      the compaction to happen before continuing. You can file a ticket with 
+
+      If you are enabling compaction for the first time, you will need to wait for
+      the compaction to happen before continuing. You can file a ticket with
       support if you want to ensure compaction has completed.
 
 #. Perform a Rolling Resync
 
-   Perform a rolling resync of your Atlas cluster, a process explained in 
+   Perform a rolling resync of your Atlas cluster, as described in
```

Comment:
This is just a nitpick, but it seems irrelevant to link out to this documentation if people can't actually perform this step themselves. Instead, I think we should make this step in ln 44 above "File a support ticket" to have a rolling resync done for your Atlas cluster.